junebug/tests/test_logging_service.py

```diff
@@ -207,6 +210,20 @@ def test_start_stop(self):
         self.assertFalse(self.service.registered())
         self.assertEqual(self.service.logfile.closed_count, 1)
 
+    @inlineCallbacks
+    def test_dir_create(self):
+        '''If log directory already exists, make sure it is not recreated'''
+        if not os.path.exists(self.service.path):
+            os.makedirs(self.service.path, 0777)
+            self.stat1 = os.stat(self.service.path)
+            yield self.service.startService()
+            self.stat2 = os.stat(self.service.path)
+            self.assertEqual(self.stat1, self.stat2)
+            yield self.service.stopService()
+        yield self.service.startService()
+        self.assertTrue(os.path.exists(self.service.path))
+        yield self.service.stopService()
+
```

Comment:
We probably also want a test to make sure that if the directory doesn't exist, that it is actually created when we start the service.